doc/ref/configuration/master.rst

```diff
@@ -1129,6 +1129,22 @@ a minion performs an authentication check with the master.
 
     auth_events: True
 
+.. conf_master:: auth_events_pend_autosign_grains
+
+``auth_events_pend_autosign_grains``
+---------------
```

Comment:
This underline needs to be the full length of the title, to include the backticks